CHANGELOG.md

```diff
@@ -42,8 +42,9 @@ This version comes with a variety of technical changes that might affect the fun
   ```js
   // v5
   await browser.url('https://duckduckgo.com/')
-  const elem = await browser.$('#search_form_input_homepage')
+  const elem = await $('#search_form_input_homepage')
```

Comment:
Can we revert this change? The example script here shows the standalone version. We could keep it like this but then we should remove all `await`s